ONSdigital/eq-survey-runner
# Update linting rules to catch a range of Python issues

### What is the context of this PR?

This PR adds extra linting rules to our CI setup to help us catch common errors or causes of bugs in the way in which we write python code.  It is influenced by some of the approaches taken by Mozilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1237977
### How to review
1. Check out the branch
2. Run `pip install -r requirement.txt` to install new flake8 plugins
3. Run `./scripts/run_tests.sh` to execute the Flake8 code linting
4. Check that they all pass.
### Who worked on the PR

@dhilton
